docs/blog/newsletter-march-2018.rst

```diff
@@ -0,0 +1,95 @@
+
+- Writing is an ‘endemic skill,’ meaning it’s practically universal, to at least a basic degree. That can make it challenging for people to recognize mastery because, on the surface, it's hard to distinguish mastery from what everyone else is doing.
+
+- This underlying, "anyone can write" assumption gets particularly troublesome when used to justify the opinion that tech writers' chief value is that they free up engineers to write code.
+
+- Technical writers’ knowledge gaps are front-and-center because they typically rely on subject matter experts to confirm what they write, as well as to provide feedback and corrections.
+
+- Technical writing has historically been female-dominated, and even the most accomplished women sometimes rely on self-deprecating language to shield themselves from criticism.
+
+- Technical writing roles don’t always have clear lines of specialization like other roles in the tech environment.
+
+And that’s just a thin summary of this rich conversation! Although we don't usually share out the transcripts of the conversations we highlight, this one so clearly struck a chord that we thought we'd save – and anonymize – the whole thing, It's available in `this gist <https://gist.github.com/mjang/c49474fb7bbccbf06a9f47e7db096456>`_, if you want to read the whole discussion, including some ideas for impostor syndrome relief.
+
+--------------------------------------------
+Docs-as-code and its discontents: Versioning
+--------------------------------------------
+
+A particularly in-depth exploration of content versioning came up this month, which we thought was worth sharing. The question that sparked the conversation was specifically about how and whether folks versions their docs content separately from their overall documentation site. From there a discussion evolved around managing multiple versions of docs with a static site generator (SSG).
+
+One approach that was suggested was `this Jekyll-based versioning solution <https://github.com/justwriteclick/versions-jekyll>`_ `This discussion  of the CoreOS docs <https://github.com/coreos/docs/issues/1082>`_ was also cited as a useful resource. Someone else suggested using git submodules to fetch different versions. But eventually most folks agreed: SSGs are not the optimal solution if you need to serve multiple versions of your content.
+
+Suggestions for other solutions included:
+
+* `Sphinx <http://www.sphinx-doc.org/en/master/>`_
+* `DocFX <https://dotnet.github.io/docfx/>`_
+* `Antora <https://antora.org/>`_ (an Asciidoctor toolchain)
+* `MadCap Flare <https://www.madcapsoftware.com/products/flare/>`_
+
+---------------------
+Featured job postings
+---------------------
+
+*To post or apply for a job, visit the `Write the Docs job board <https://jobs.writethedocs.org/>`_.*
+
+`Looking for a Documentarian in Berlin <https://jobs.writethedocs.org/job/50/looking-for-a-documentarian-in-berlin/>`_
+ NetworkedAssets GmbH, Full-time
+
```

Comment:
Dumping the tagging writeup here in case it helps:
-----------------------------------
The whys and wherefores of tagging docs
-----------------------------------

A question about topical tagging strategy produced both some useful ideas for strategy and the inevitable reminder that maintaining doc sets over time is challenging. Some folks wanted to provide tags to integrate with search, to provide more sophisticated results than exact string matching. Others saw tagging as little more than help for SEO. The original poster wanted a system to check for consistency across doc sets, in which tags in metadata could be correlated both within a single doc set and across multiple doc sets. Topics with shared tags could then be checked to make sure that information complements and doesn't compete or conflict across topics.

Most people who had tried tagging, though, agreed that it was difficult to maintain consistent tagging over time, even with small content sets. Automated tagging tends to produce too many tags. Controlled terminology seems like part of the answer, but it's also a huge project to maintain. Conclusion: you need a big compelling reason to undertake the long-term maintenance that a really useful tagging strategy requires. But in the right circumstances, it could help maintain clarity and consistency across multiple large doc sets.